src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleaseChecklistService.cs

```diff
@@ -107,7 +108,10 @@ public async Task<List<ReleaseChecklistIssue>> GetErrors(Release release)
                 errors.Add(new ReleaseChecklistIssue(ValidationErrorMessages.PublicMetaGuidanceRequired));
             }
 
-            if (release.Amendment && release.Updates.All(update => update.ReleaseId != release.Id))
+            if (release.Amendment
+                && !release.Updates.Any(update =>
+                   update.ReleaseId == release.Id
+                   && update.Created > release.Created))
```

Comment:
Done. I've also renamed the arguments of AssertAmendedUpdateCorrect - not because they were bad, but because I tripped up on it and it is now absolutely clear.